tests/jenkins/TestRunIntegTestScript.groovy

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import jenkins.tests.BuildPipelineTest
+import org.junit.Before
+import org.junit.Test
+
+
+class TestRunIntegTestScript extends BuildPipelineTest {
+
+    @Test
+    public void TestRunIntegTestScript() {
```

Comment:
Can we add a lib-tester for the library?  Reference - https://github.com/opensearch-project/opensearch-build/blob/main/DEVELOPER_GUIDE.md#tests-for-jenkins-libraries